### PR TITLE
Remove duplication of names from github actions matrix

### PR DESCRIPTION
**Pull request by BenjaminW3**
Include section does not need to match matrix in any way anymore.
See: https://github.blog/changelog/2020-04-15-github-actions-new-workflow-features/

### CODE REVIEW

**Comment by BenjaminW3** [comment]
The failing build is unrelated. PR can be merged after approval.

**Review by sbastrakov — APPROVED** [review]

**Pull request merged**
